Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp

```diff
@@ -73,6 +73,15 @@ static void wipeAlphaChannelFromPixels(std::span<uint8_t> pixels)
 }
 #endif
 
+static inline const Vector<const void*> asPointers(std::span<const GCGLsizei> offsets)
```

Comment:
The `const` here in `const Vector` is not helpful and should probably be removed

Also for code like this we can also consider returning `FixedVector` for slightly better efficiency, if we had a `WTF::map` that supported; no particular reason to prefer `Vector` with its slightly larger size and capability to be efficiently resized after creation